src/alertsmanagement/azext_alertsmanagement/_params.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+
+    name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
+
+    action_rule_name = CLIArgumentType(overrides=name_arg_type, help='Name of action rule.',
+                                       id_part='name')
+
+    with self.argument_context('alertsmanagement action-rule create') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('name', action_rule_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('tags', tags_type)
+        c.argument('status', arg_type=get_enum_type(['Enabled', 'Disabled']), id_part=None, help='Indicates if the given action rule is enabled or disabled')
+        c.argument('rule_type', arg_type=get_enum_type(['Suppression', 'ActionGroup', 'Diagnostics']), help='Indicates type of action rule')
+        c.argument('description', help='Description of action rule')
+        c.argument('scope_type', help='Type of target scope', arg_type=get_enum_type(['ResourceGroup', 'Resource']))
+        c.argument('scope', nargs='+', help='List of ARM IDs (space-delimited) of the given scope type which will be the target of the given action rule.')
+        c.argument('severity', nargs='+', help='Filter alerts by severity. All filters should follow format "operator value1 value2 ... valueN". Operator is one of Equals, NotEquals, Contains and DoesNotContain.')
+        c.argument('monitor_service', nargs='+', help='Filter alerts by monitor service')
+        c.argument('monitor_condition', nargs='+', help='Filter alerts by monitor condition')
+        c.argument('target_resource_type', nargs='+', help='Filter alerts by target resource type')
+        c.argument('alert_rule_id', nargs='+', help='Filter alerts by alert rule ID')
```

Comment:
alert_rule to support both name and id?

Reply:
parameters that end with `-id` are supposed to use GUID. If it's not GUID, probably rename the option.

Reply:
Done.
Done.